molgenis-metadata-manager/src/main/frontend/src/store/actions.js

```diff
@@ -32,7 +32,12 @@ export default {
     get({apiUrl: '/api'}, '/v2/sys_md_EntityType?num=10000')
       .then(response => {
         const nonSystemEntities = response.items.filter(function (item) {
-          return !item.package.id.startsWith('sys')
+          // TODO we should filter on parent package is sys, a user can also select a package name starting with sys_
```

Comment:
![screen shot 2017-06-12 at 08 36 59](https://user-images.githubusercontent.com/10075439/27021693-590063ac-4f4a-11e7-84b4-d09e0aa09d5c.png)
I mean it is possible to create a package with the name sys_something (see screenshot). If a user creates a package with such a name, it will still be filtered out now, while it is not a system package.